### DISCUSSION TITLE
jenkins frontend setup

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1965] [Q&A] [answered]
![Screenshot (61)](https://github.com/user-attachments/assets/476e99f7-d941-49b5-babb-4e34660ee40f)

**Comment by chaithanya1812** [comment, marked as answer]
## cross-check once this is your frontend-ip??
![image](https://github.com/user-attachments/assets/2de90cf5-7a43-4d8d-a16d-e509811da0ff)

**Comment by Rezanmonis** [comment]
We are looking for frontend and backend developers as our teammates for our edtech project rezanmonis@gmail.com
